src/includes/blogpost.njk

```diff
@@ -24,7 +24,7 @@
       <div class="blogpost-meta-share-twitter">
         <dt class="share-twitter">Share</dt>
         <dd class="share-twitter">
-          <a class="share-twitter-link" href="https://twitter.com/intent/tweet/?text={{ tweetText }}&url={{ tweetURL }}?{{ 'utm_source=twitter&utm_medium=social&utm_campaign=shares_from_blog' | urlencode }}">
+          <a class="share-twitter-link" href="https://twitter.com/intent/tweet/?text={{ tweetText }}&url={{ tweetURL }}?{{ 'utm_source=twitter&utm_medium=social&utm_campaign=amo_blog_share' | urlencode }}">
```

Comment:
Should UTM params be covered by a test?

Reply:
Yeah, we could using the Wordpress test suite probably. I am also thinking about having a test suite for the generated content of the blog, but I am not sure how to write that given that content won't be fixed (even knowning the URL of a blogpost is not easy)

Reply:
I added a php assertion for now